cli/internal/helm/loader.go

```diff
@@ -644,3 +474,18 @@ func loadChartsDir(efs embed.FS, dir string) (*chart.Chart, error) {
 
 	return loader.LoadFiles(files)
 }
+
+func controlPlaneTolerations() []map[string]any {
```

Comment:
Could this be a variable? I think this could even be scoped to just the function where it is used currently (if there are no plans to re-use this in the near future of course).

Reply:
Changed to variable and moved to values.go